docs/content/1.get-started/5.migration.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Migration to v2
+description: 'A comprehensive guide to migrate your application from Nuxt Image v1 to Nuxt Image v2.'
+navigation.title: 'Migration'
+---
+
+Nuxt Image v2 brings improved performance, enhanced TypeScript support, and a better developer experience.
+
+Most apps should be able to upgrade with minimal changes.
+
+## Update dependencies
+
+Update `@nuxt/image` to v2:
+
+::code-group{sync="pm"}
+
+```bash [pnpm]
+pnpm add @nuxt/image
+```
+
+```bash [yarn]
+yarn add @nuxt/image
+```
+
+```bash [npm]
+npm install @nuxt/image
+```
+
+```bash [bun]
+bun add @nuxt/image
+```
+
+::
+
+## Check your Nuxt version
+
+Nuxt Image v2 requires at least **Nuxt 3.1**. Check your current version:
+
+```bash
+npm list nuxt
+```
+
+If you're on Nuxt 3.0.x, upgrade first:
+
+```bash
+npx nuxt upgrade --channel v3
+```
+
+::tip
+Nuxt Image v2 is fully compatible with Nuxt 4. See the [Nuxt 4 migration guide](https://nuxt.com/docs/getting-started/upgrade#nuxt-4) if you're ready to upgrade.
+::
+
+## Update screen sizes
+
+The `xs` and `xxl` breakpoints have been removed to align with Tailwind CSS defaults.
+
+Search for usage in your project:
+
+```bash
+grep -r "sizes.*xs:" --include="*.vue"
+grep -r "sizes.*xxl:" --include="*.vue"
+```
+
+**If you are using these screen sizes**, you can either replace `xs` with `sm` and `xxl` with `2xl`:
```

Comment:
This holds for xxl, but not for xs.

See https://github.com/nuxt/image/pull/1931/files#diff-febc21f95a6b1803f5a608291db3659113139a4f37e9b58afa91cd9d9bfa600fL69-L75